_accounts.md

```diff
@@ -111,3 +111,50 @@ Retrieves the details about a specific account.
 ### Response
 
 Returns a fully formed [Account Object](#account-object) representing the requested account.
+
+## Adding credit/debit card accounts
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox. These can be used to add card accounts to an Uphold wallet which will then allow for the testing of transactional flows.
+
+<aside class="notice">
+  Any value entered for <i>expiry date</i> and <i>CVV</i> will be accepted as valid when used in conjunction with the card data listed in this section.
+</aside>
+
+### Approved cards
+
+The following is a list of valid card data that can be used to add a card account to an Uphold wallet for testing purposes.
+
+Number           | Brand      | Type   | Country | OCT settlement |
+---------------- | ---------- | ------ | ------- | -------------- |
+5355223761921186 | mastercard | debit  | GB      | instant        |
+5573606426146833 | mastercard | debit  | GB      | instant        |
+5318773012490080 | mastercard | debit  | US      | instant        |
```

Comment:
This one is not on the list here https://github.com/uphold/credit-card-gateway/pull/895/files.

Reply:
It's present on the document that I was pointed to as the source of truth for this updated.
<img width="1260" alt="image" src="https://github.com/uphold/docs/assets/3464842/7f9c6585-790b-482e-8c0f-00485a8dd538">

Reply:
Should we add it to the whitelisted cards?

Reply:
That's entirely up to you guys. Either that, or it should be removed from that table on the repository.

Reply:
Updated with the latest information in the PR you referenced offline.